### ISSUE TITLE
About Waymo datasets preprocess and the evaluation

### ISSUE THREAD

**Issue by Worker789** [hailanyi/CasA#19]
Hello, I have some questions about the preprocess of the Waymo datasets and the evaluation. I use the official code to generate the datasets and when I train the model, some information is as follows, is that right?

INFO  Database filter by min points Vehicle: 175529 =>159486
INFO  Database filter by min points Pedestrian: 63325 => 51739
INFO  Database filter by min points Cyclist: 21442 => 19535
INFO  Database filter by difficulty Vehicle: 159486 => 159486
INFO  Database filter by difficulty Pedestrian: 51739 => 51739
INFO  Database filter by difficulty Cyclist: 19535 => 19535

I used 20% training dataset to train, during training I changed the **MAX_NUMBER_OF_VOXELS** to **150000**, **BATCH_SIZE_PER_GPU** to **1**, when I evaluated the model on the whole validation set of Waymo with 29 epoch, the metrics of the recall are as follows:

INFO  recall_roi_0.3: 0.385299
INFO  recall_rcnn_0.3: 0.374685
INFO  recall_roi_0.5: 0.308684
INFO  recall_rcnn_0.5: 0.302918
INFO  recall_roi_0.7: 0.169099
INFO  recall_rcnn_0.7: 0.168316
INFO  Average predicted number of objects(39987 samples): 380.692

I'd like to know if I'm having problems in preprocessing the dataset, or if I shouldn't change any settings. Hope to get your reply!

**Comment by hlh2001** [comment]
I meet the same problem, do you know how to get the right training results? Thank you!
